src/status_im/keycard/core.cljs

```diff
@@ -528,3 +529,17 @@
   {:events [:keycard.callback/on-register-card-events]}
   [{:keys [db]} listeners]
   {:db (update-in db [:keycard :listeners] merge listeners)})
+
+(defn onboarding-intro-back-handler
+  "The back button handler is used to manage device back press.
+
+  If the handler returns false, the back button functions as usual (ie. dispatchs GO_BACK event).
+  If it returns true, the back button becomes inactive.
+
+  We want to deactivate the back button when the user comes from key-storage and migration flow."
+  []
+  (-> re-frame.db/app-db
+      deref
```

Comment:
just wondering why do you prefer `deref` under the `@` ?

Reply:
No particular reason, I just wrote it while I was testing it out. Have replaced `deref` with `@`. That's more crisp.